proof/refine/RISCV64/Refine.thy

```diff
@@ -1268,37 +1235,27 @@ lemma ckernel_invariant:
      apply (rule hoare_weaken_pre)
       apply (rule kernelEntry_invs')
      apply clarsimp
+     apply (frule_tac a=sa and b=s in curthread_relation)
```

Comment:
Does this one have rename_tac for `sa` and `s`  before? (I didn't spot any, but it's hard to see in the GitHub interface). I've just had a bunch of these fail for Isabelle2025..

Reply:
Oops, added now.